proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -42,24 +44,26 @@ message CreateWebhookRequest {
 
 // WebhookEvent contains the currently supported webhook event types.
 enum WebhookEvent {
-  // WEBHOOK_EVENT_UNSPECIFIED will result in a noop for registering webhooks.
+  // WEBHOOK_EVENT_UNSPECIFIED is a safe noop default for webhook events
+  // subscription. It will trigger an error if trying to register a webhook with
+  // this event.
   WEBHOOK_EVENT_UNSPECIFIED = 0;
-  // WEBHOOK_EVENT_REPOSITORY_PUSH is emitted whenever a successful buf push
-  // is completed for a specific repository.
+  // WEBHOOK_EVENT_REPOSITORY_PUSH is emitted whenever a successful buf push is
+  // completed for a specific repository.
   WEBHOOK_EVENT_REPOSITORY_PUSH = 1;
 }
 
-// CreateWebhookResponse is the proto repsonse representation
+// CreateWebhookResponse is the proto response representation
 // of a webhook request.
 message CreateWebhookResponse {
-  // For the user to unsubscribe from the webhook event.
-  string webhook_subscription_id = 1;
+  // Created webhook subscription.
+  SubscribedWebhook webhook = 1;
```

Comment:
Could we potentially rename the `SubscribedWebhook` message to just `Webhook`? Its refferred to as webhook in all fields and comments now, feels a little off. The only thing you'd create or delete or list would be subscribed webhooks.